src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/MFD/CJ4_MFD.html

```diff
@@ -272,17 +272,17 @@
                             <div>TAS</div>
                             <div class="Modifiable" id="Tas_Value">000</div>
                         </div>
-                        <div id="Rat">
+                        <div id="Rat" style="min-width: 112px">
```

Comment:
Inline styles because of flexbox calculation timing, won't work when moved into existing css definitions